src/main/java/org/eaa690/aerie/service/CommunicatorService.java

```diff
@@ -0,0 +1,16 @@
+package org.eaa690.aerie.service;
+
+import org.eaa690.aerie.communication.MessageSender;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public abstract class CommunicatorService {
```

Comment:
I'm guessing this class is intended to be a base class to aggregate common objects between the Email, SMS, and Slack services...  however there really isn't all that much to aggregate, other than the PropertyService.  I'd simple leave the Email, SMS, and Slack services as-is, but have each implement the MessageSender interface.